crop-mask-example/data/report.txt

```diff
@@ -1,19 +1,17 @@
 DATASET REPORT (autogenerated, do not edit directly)
 
-geowiki_landcover_2017 (Timesteps: 24)
+GeowikiLandcover2017 (Timesteps: 24)
 ----------------------------------------------------------------------------
 eo_data_complete          34270
 eo_data_export_failed       841
 eo_data_missing_values      379
-eo_data_skipped             376
```

Comment:
These are the 0.5 class probability values that are thrown out in `load_labels()` now